.github/workflows/tests.yaml

```diff
@@ -0,0 +1,60 @@
+name: Pubs tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 8 * * *'
+
+
+jobs:
+  unit-test:
+    name: Run unit tests
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
We are not testing Python 2.7 anymore. Should be start updating the readme to remove mention of Python 2 support, and then actually remove it in a future release?

Reply:
Actually the readme already states that it is not supported any more as we had already stopped testing for it some time ago. We could indeed remove all mentions of it with the next release. I will leave it for now anyway.